### ISSUE TITLE
Plugin local_wstemplate does not declare valid $plugin->component in its version.php

### ISSUE THREAD

**Issue by digicharan** [moodlehq/moodle-local_wstemplate#5]
Hello,
When I am trying to install this in my Moodle. I am getting following error. I am using Moodle 3.0.3..

Plugin local_wstemplate does not declare valid $plugin->component in its version.php.
line 328 of /lib/classes/plugin_manager.php: call to debugging()
line 348 of /lib/classes/plugin_manager.php: call to core_plugin_manager->load_present_plugins()
line 217 of /lib/classes/plugininfo/base.php: call to core_plugin_manager->get_present_plugins()
line 143 of /lib/classes/plugininfo/base.php: call to core\plugininfo\base->load_disk_version()
line 97 of /lib/classes/plugininfo/base.php: call to core\plugininfo\base::make_plugin_instance()
line 408 of /lib/classes/plugin_manager.php: call to core\plugininfo\base::get_plugins()
line 368 of /lib/classes/plugin_manager.php: call to core_plugin_manager->get_plugins_of_type()
line 758 of /lib/classes/plugin_manager.php: call to core_plugin_manager->get_plugins()
line 668 of /admin/index.php: call to core_plugin_manager->all_plugins_ok()

**Comment by mudrd8mz** [comment]
Yeah, the code here seems to be a bit outdated. The `$plugin->component` become mandatory. Pull requests will be appreciated.

**Comment by digicharan** [comment]
Thank you
